content/blog/2023/nextflow-with-gbatch.md

```diff
@@ -0,0 +1,290 @@
+title=Get started with Nextflow on Google Cloud Batch
+date=2023-01-27
+type=post
+description=We've talked about deploying Nextflow pipelines with Google Cloud Batch in the past. Join us again for an extended and updated version
+image=img/gbatch_extended.jpg
+tags=nextflow,google,cloud
+status=published
+author=Marcel Ribeiro-Dantas
+icon=marcel.png
+~~~~~~
+
+[We have talked about Google Cloud Batch before](https://www.nextflow.io/blog/2022/deploy-nextflow-pipelines-with-google-cloud-batch.html). Not only that, we were proud to announce Nextflow support to Google Cloud Batch right after it was publicly released, back in July 2022. How amazing is that? But we didn't stop there! The [Nextflow official documentation](https://www.nextflow.io/docs/latest/google.html) also provides a lot of useful information on how to use Google Cloud Batch as the compute environment for your Nextflow pipelines. Having said that, feedback from the community is valuable, and we agreed that in addition to the documentation, teaching by example, and in a more informal language, can help many of our users. So, here is a tutorial on how to use the Batch service of the Google Cloud Platform with Nextflow 🥳
+
+### Running an RNAseq pipeline with Google Cloud Batch
+
+Welcome to our RNAseq tutorial using Nextflow and Google Cloud Batch! RNAseq is a powerful technique for studying gene expression and is widely used in a variety of fields, including genomics, transcriptomics, and epigenomics. In this tutorial, we will show you how to use Nextflow, a popular workflow management tool, to perform an RNAseq analysis on Google Cloud Batch, a scalable cloud-based computing platform.
+
+Nextflow allows you to easily develop, execute, and scale complex pipelines on any infrastructure, including the cloud. Google Cloud Batch enables you to run batch workloads on Google Cloud Platform (GCP), with the ability to scale up or down as needed. Together, Nextflow and Google Cloud Batch provide a powerful and flexible solution for RNAseq analysis.
```

Comment:
it's worth to point out that's a proof-of-concept RNAseq pipeline. For a real one they should look at https://github.com/nf-core/rnaseq